src/rotation.lua

```diff
@@ -246,23 +246,11 @@ function SilentRotate:isHunterRegistered(GUID)
 end
 
 -- Return our hunter object from name or GUID
-function SilentRotate:getHunter(name, GUID)
+function TranqRotate:getHunter(searchTerm)
 
-    -- Known limitation: if both name and GUID are set but do not correspond to the same unit, results are undefined
-
-    if (GUID ~= nil) then
-        for key,hunter in pairs(SilentRotate.hunterTable) do
-            if (hunter.GUID == GUID) then
-                return hunter
-            end
-        end
-    end
-
-    if (name ~= nil) then
-        for key,hunter in pairs(SilentRotate.hunterTable) do
-            if (hunter.name == name) then
-                return hunter
-            end
+    for _, hunter in pairs(TranqRotate.hunterTable) do
```

Comment:
The table should probably be SilentRotate.hunterTable instead of TranqRotate.hunterTable